src/languages/en.ts

```diff
@@ -951,7 +951,7 @@ const translations = {
         posted: 'Posted',
         deleteReceipt: 'Delete receipt',
         deletedTransaction: ({amount, merchant}: DeleteTransactionParams) => `deleted an expense on this report, ${merchant} - ${amount}`,
-        movedTransaction: ({reportUrl, reportName}: MovedTransactionParams) => `moved this expense to <a href="${reportUrl}">${reportName}</a>`,
+        movedTransaction: ({reportUrl, reportName}: MovedTransactionParams) => `moved this expense to ${reportName ? `<a href="${reportUrl}">${reportName}</a>` : '[deleted]'}`,
```

Comment:
Should it say "inaccessible" report rather than deleted? May be because it could be the report that is unshared from the user, no?

Reply:
Sorry, I should have given more context. I had discussed this 1:1 with @JmillsExpensify. We're going with `[deleted]`